src/sbp.c

```diff
@@ -331,6 +332,23 @@ int _write(int file, char *ptr, int len)
   }
 }
 
+/** Directs log_ output to the SBP log message */
+void log_(u8 level, const char *msg, ...)
+{
+  msg_log_t *log;
+  va_list ap;
+  char buf[256];
```

Comment:
The named constant isn't in libsbp yet - will move over to the constant once it comes in.